lib/run-editor-cmd.js

```diff
@@ -16,4 +16,7 @@ module.exports = function({ editor, filePath, line, column }) { //include any ot
   if (editor === 'vim') {
     execSync(`vim ${filePath} "+call cursor(${line}, ${column})"`);
   }
+  if (editor === 'emacs') {
+    execSync(`emacs +${line}:${column} ${filePath}`);
+  }
```

Comment:
@ctjhoa Thanks for raising PR.

One quick question. Is it this supposed to be `emacsclient`

Quoting from https://www.emacswiki.org/emacs

> EmacsClient allows one to open a file for editing in an already running Emacs. Because it doesn't start a new Emacs instance at each invocation, you can set up EmacsClient as the default editor, e.g. in the environment variable EDITOR or VISUAL.

Reply:
@siva-sundar I'm ok with both. I was worried about having a client without daemon running.
According to https://www.emacswiki.org/emacs/EmacsClient#toc4
> Specifying -a=”” tells emacsclient to start an an instance of emacs if it cannot find one already running.

So I will fixup my commit with that